definitions/npm/moment_v2.x.x/test_moment-v2.js

```diff
@@ -2,38 +2,175 @@
+describe('Parse, moment()', () => {
+  const m3: moment = moment([123, 123]);
+  // https://momentjs.com/docs/#/parsing/now/
+  moment();
+  moment([]);
+  moment({});
 
-// $ExpectError
-moment.unix("1234");
+  moment('2015-01-01');
+  moment("12-25-1995", "MM-DD-YYYY");
+  moment('It is 2012-05-25', 'YYYY-MM-DD', true);
+  // https://momentjs.com/docs/#/parsing/string-formats/
+  moment("12-25-1995", ["MM-DD-YYYY", "YYYY-MM-DD"]);
+  moment("29-06-1995", ["MM-DD-YYYY", "DD-MM-YYYY"], 'fr');
+  moment("29-06-1995", ["MM-DD-YYYY", "DD-MM-YYYY"], true);
+  moment("05-06-1995", ["MM-DD-YYYY", "DD-MM-YYYY"], 'fr', true);
+  // $ExpectError
+  moment("05-06-1995", ["MM-DD-YYYY", "DD-MM-YYYY"], 'fr', 1);
+  // https://momentjs.com/docs/#/parsing/object/
+  moment({ hour: 15, minute: 10 });
+  moment({ y: 2010, M: 3, d: 5, h: 15, m: 10, s: 3, ms: 123 });
+  moment({
+    year: 2010,
+    month: 3,
+    day: 5,
+    hour: 15,
+    minute: 10,
+    second: 3,
+    millisecond: 123
+  });
+  moment({
+    years: 2010,
+    months: 3,
+    days: 5,
+    hours: 15,
+    minutes: 10,
+    seconds: 3,
+    milliseconds: 123
+  });
+  moment({
+    years: 2010,
+    months: 3,
+    date: 5,
+    hours: 15,
+    minutes: 10,
+    seconds: 3,
+    milliseconds: 123
+  });
+  moment({
+    years: "2010",
+    months: "3",
+    date: "5",
+    hours: "15",
+    minutes: "10",
+    seconds: "3",
+    milliseconds: "123"
+  }); // from 2.11.0
+  // $ExpectError hour is invalid type
+  moment({ hour: null, minute: 10 });
+
+  // https://momentjs.com/docs/#/parsing/array/
+  moment([2015, 0]); // This would equal 2015-01-01
+  // $ExpectError only numbers are valid for Array API
+  moment(["2015"]);
+  // $ExpectError needs a format string before the strictness flag
+  moment('2015-01-01', true);
+
+  // $ExpectError only string values can have formatting parameters
+  moment([2015, 0], "MM-DD-YYYY");
+  // $ExpectError
+  moment([2015, 0], "MM-DD-YYYY", true);
+  // $ExpectError
+  moment([2015, 0], moment.ISO_8601);
+  // $ExpectError
+  moment({ hour: 15, minute: 10 }, "MM-DD-YYYY");
+  // // https://momentjs.com/docs/#/parsing/special-formats/
+  moment("2010-01-01T05:06:07", moment.ISO_8601);
+  moment("2010-01-01T05:06:07", ["YYYY", moment.ISO_8601]);
+  // https://momentjs.com/docs/#/parsing/unix-timestamp-milliseconds/
+  moment(1318781876406);
+  // https://momentjs.com/docs/#/parsing/date/
+  moment(new Date());
+  // https://momentjs.com/docs/#/parsing/moment-clone/
+  moment(moment());
+});
+
+// https://momentjs.com/docs/#/parsing/unix-timestamp/
+describe('Parse, moment.unix()', () => {
+  moment.unix(123);
+  // $ExpectError
+  moment.unix("1234");
+  // $ExpectError
+  moment.unix(undefined);
+  // $ExpectError
+  moment.unix();
+});
+
+// https://momentjs.com/docs/#/parsing/utc/
+// Basically all the same things as moment(), as the docs say:
+// "If you want to parse or display a moment in UTC, you can use moment.utc() instead of moment()."
```

Comment:
There were more, working ways to `moment.utc()` than was immediately obvious from looking at https://momentjs.com/docs/#/parsing/utc/. I figured the comment

> If you want to parse or display a moment in UTC, you can use `moment.utc()` instead of `moment()`.

was the defining design decision here, so I mostly copy-pasted these specs from the `moment()` specs.